src/HydraAuction/Runner.hs

```diff
@@ -69,8 +88,8 @@ data ExecutionContext = MkExecutionContext
   , actor :: !Actor
   }
 
-{- | Hydra computation executor. Note that @Runner@ is
- de facto `ReaderT ExecutionContext IO`.
+{- | HydraAuction specific L1 computation executor.
```

Comment:
I like these changes, maybe we could rename runner to something like `CardanoRunner`, just to make it clear this is operating on L1

Reply:
Yep, I thought about this.